crypto/bn254/utils.go

```diff
@@ -145,30 +143,3 @@ func DeserializeG2(b []byte) *bn254.G2Affine {
 	p.Y.A1.SetBytes(b[96:128])
 	return p
 }
-
-func MakePubkeyRegistrationData(
```

Comment:
where has these functions moved? or it's not needed?

Reply:
yeah not needed anymore, this was the old way of registering.
Forget the details but no longer needed.
Everything that's needed for registration is in https://github.com/Layr-Labs/eigensdk-go/blob/85d637f48166edc79158446b02670ae8caceae88/chainio/clients/avsregistry/writer.go#L163
I left a TODO to clean and refactor that function eventually :D